### PR TITLE
Adding Clickpipes docs

### PR DESCRIPTION
**Pull request by Ryado**

### CODE REVIEW

File: docs/en/integrations/data-ingestion/clickpipes/index.md [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+## Setup
+
+### 1. Enable ClickPipes for your cloud organization
+
+ClickPipes is currently accessible in private preview. You can join our waitlist by filling [this form](https://clickhouse.com/cloud/clickpipes#joinwaitlist). Please note that during the Private Preview phase, ClickPipes is available only for Amazon Web Services backed services, in the `us-east-2` region.
+
+
+### 2. Creating your first ClickPipe
+
+1. Access the SQL Console for your ClickHouse Cloud Service running in AWS `us-east-2` region.
```
Comment (mshustov) [inline docs/en/integrations/data-ingestion/clickpipes/index.md:36]: Should we mention that we recommend creating a new instance? Even if it's mentioned below

**Comment by robot-clickhouse-ci-2** [comment]
This is an automated comment for commit a7f698322567cc3ba70ff5cb65e6610d2923f1c7 with description of existing statuses. It's updated for the latest CI running
The full report is available [here](https://s3.amazonaws.com/clickhouse-test-reports/1330/a7f698322567cc3ba70ff5cb65e6610d2923f1c7/ci_running.html)
The overall status of the commit is 🟢 success

<table><thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody><tr><td>Docs Check</td><td>Builds and tests the documentation</td><td>🟢 success</td></tr>
<tr><td>Vercel</td><td>There's no description for the check yet, please add it to tests/ci/ci_config.py:CHECK_DESCRIPTIONS</td><td>🟢 success</td></tr>
</table>

File: docs/en/integrations/data-ingestion/clickpipes/index.md [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+<img src={require('./images/cp_step4a3.png').default} class="image" alt="ClickPipes Illustration"/>
+<br/>
+
+6. Alternatively, you can decide to ingest your data in an existing ClickHouse table. In that case, the UI will allow you to map fields from Kafka with the ClickHouse fields in the selected destination table.
+
+<br/>
+<img src={require('./images/cp_step4b.png').default} class="image" alt="ClickPipes Illustration"/>
+<br/>
+
+7. Finally, you can decide to enable the error logging table. When enabled, ClickPipes will create a table next to your destination table with the postfix `_error`. This table will contain any errors from the operations of your ClickPipe (network, connectivity, etc.) and also any data that don't conform to the schema specified in the previous screen. The error table has a [TTL](https://clickhouse.com/docs/en/engines/table-engines/mergetree-family/mergetree#table_engine-mergetree-ttl) of 7 days.
```
Comment (mshustov) [inline docs/en/integrations/data-ingestion/clickpipes/index.md:89]:
```suggestion
7. Finally, you can decide to enable the error logging table. When enabled, ClickPipes will create a table next to your destination table with the postfix `_clickpipes_error`. This table will contain any errors from the operations of your ClickPipe (network, connectivity, etc.) and also any data that don't conform to the schema specified in the previous screen. The error table has a [TTL](https://clickhouse.com/docs/en/engines/table-engines/mergetree-family/mergetree#table_engine-mergetree-ttl) of 7 days.
```

**Review by mshustov — APPROVED** [review]

File: docs/en/integrations/data-ingestion/clickpipes/index.md (hunk 1 above)
Comment (Ryado) [reply docs/en/integrations/data-ingestion/clickpipes/index.md:36]: The mention below should be enough

**Pull request merged**